metadata.json

```
@@ -1,6 +1,6 @@
 {
     "shell-version": [
-        "46"
+        "47"
```

If the code works with both versions, then why not use both?

```suggestion
        "46", "47"
```